crates/sui-bridge/src/eth_syncer.rs

```diff
@@ -94,7 +94,7 @@ where
             interval.tick().await;
             let Ok(Ok(new_value)) = retry_with_max_elapsed_time!(
                 eth_client.get_last_finalized_block_id(),
-                time::Duration::from_secs(10)
+                time::Duration::from_secs(600)
```

Comment:
Will this cause the time from deposit to approved to increase potentially?

Reply:
no it won't. If we pass this duration, we would `continue` anyway. This change is to increase the latency between each query to avoid hammer the rate limited eth node